sources/cloudwatch_log_group/cloudwatch_log_group_source.go

```diff
@@ -77,7 +77,7 @@ func (s *AwsCloudWatchLogGroupSource) Identifier() string {
 
 func matchesAnyPattern(target string, patterns []string) bool {
 	for _, pattern := range patterns {
-		match, err := filepath.Match(pattern, target)
+		match, err := path.Match(pattern, target)
```

Comment:
Given that we decided to not escape chars, can you please add 1-2 examples in the CW log source table doc showing how escaping chars works? This can help us determine if we like that design as well.

Reply:
Added example in CW source doc.